src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/NftRewardTierModal.tsx

```diff
@@ -38,16 +39,18 @@ export default function NftRewardTierModal({
 
   const onFormSaved = async () => {
     await nftForm.validateFields()
+    console.info('descriptioN: ', nftForm.getFieldValue('description'))
```

Comment:
**nitpick** can we get rid of console.infos pls

Reply:
My bad